packages/core/src/components/table/table-body-row-expandable/readme.md

```diff
@@ -1,6 +1,38 @@
 # tds-table-body-row-expended
 
+### Using the `:part` Selector
 
+The `:part` selector allows you to style elements inside a shadow DOM that have been exposed via the `part` attribute. This is particularly useful for customizing the appearance of web components.
```

Comment:
might wanna look into having this explanation of how to use :part somewhere else. Could be a very generic explanation for when this sort of feature is added in more and more components

Reply:
Yeah, it could be a section in the intro part of the storybook in the future. So far we had request specifically for this component, but lets see, I can see that people might ask for more of it in other components.